cypress-tests/cypress/constants/texts/postgreSql.js

```diff
@@ -0,0 +1 @@
+export const postgreSqlText = {};
```

Comment:
@emidhun I think the text file is missing here. Please add the file.

Reply:
Added the missing labels to the file.